Tools/db.json

```diff
@@ -334,6 +335,12 @@
                 "Date": "03-28-2021",
                 "Src": "Tiny-Webini-1.4-beta.tar",
                 "Windows": "Tiny-Webini-1.4-bin.rar"
+            },
+            {
+                "Name": "Tiny-Webini-1.5",
+                "Date": "05-01-2021",
+                "Src": "Tiny-Webini-1.5-beta.tar",
```

Comment:
Later source packages should be listed as .tar.gz files. It was a fluke of mine because Safari likes to auto uncompress packaged files, so it would always give me a .tar instead.